src/main/java/org/apereo/openlrs/controllers/xapi/XAPIExceptionHandlerAdvice.java

```diff
@@ -49,18 +49,18 @@
  * @author Gary Roybal, groybal@unicon.net
  */
 @ControllerAdvice(annotations = RestController.class)
-public class XAPIExceptionHandlerAdvice {
+public class XApiExceptionHandlerAdvice {
```

Comment:
I'm having some git issues with case. New one to me :( I was just about to close it. I'll reissue later.